apis/python/src/tiledbsoma/_query_condition.py

```diff
@@ -130,15 +131,17 @@ def __attrs_post_init__(self):
 
     def init_query_condition(
         self,
-        schema: tiledb.ArraySchema,
-        enum_to_dtype: dict,
+        schema: pa.Schema,
         query_attrs: Optional[List[str]],
     ):
-        qctree = QueryConditionTree(schema, enum_to_dtype, query_attrs)
-        self.c_obj = qctree.visit(self.tree.body)
+        try:
+            qctree = QueryConditionTree(schema, query_attrs)
+            self.c_obj = qctree.visit(self.tree.body)
+        except Exception as pex:
+            raise SOMAError(pex)
```

Comment:
Ditto re above: worth adding a test for if this isn't a false positive by the coverage tool